src/Compilers/CSharp/Test/Emit/Emit/EmitErrorTests.cs

```diff
@@ -339,6 +339,7 @@ public static void Main ()
             var source = builder.ToString();
             var expectedOutput = expectedOutputBuilder.ToString();
 
+            Assert.Equal("experimental-data-section-string-literals", Feature.ExperimentalDataSectionStringLiterals);
             var expectedDiagnostics = new[]
             {
                 // (15,26): error CS8103: Combined length of user strings used by the program exceeds allowed limit. Try to decrease use of string literals or try the EXPERIMENTAL feature flag 'experimental-data-section-string-literals'.
```

Comment:
Probably orthogonal to this PR, but consider adding a parameter for the feature flag name in the error message, so we also ensure it's not getting localized.